docs/features/README.md

```diff
@@ -0,0 +1,66 @@
+# Ostracon: A Fast, Secure Consensus Layer for The Blockchain of New Token Economy
+
+Version 1.0 :: [日本語](README_ja.md)
+
+## Ostracon Overview
+
+Ostracon is a core-component that provides a Byzantine fault-tolerant (BFT) consensus mechanism for the LINE Blockchain ecosystem. This determines the order of transactions that are executed by applications, then generates and verifies blocks which are containers of transactions.
+
+LINE Blockchain sets out a number of principles to be archived in selecting the technology in order to make the consensus mechanism applicable not only to services on the internet, but also to finance and industry.
+
+**Security**: Completeness and soundness sufficient for practical use, based on cryptographic theory.
+**Consistency**: A consensus algorithm with strong integrity (finality).
+**Fault-Tolerance**: Safety and liveness against system failures, including Byzantine failures.
+**Performance and Scalability**: One block every two seconds with a capability of 1000 TPS or above.
+**Inter-chain Connectivity**: interoperability with other blockchains besides LINE Blockchain.
+
+P2P consensus algorithms based on BFT are more suitable than Bitcoin-like proof of work (PoW) in terms of functionality and performance. Among them, Tendermint-BFT, with its modern blockchain-optimized design, was the closest implementation in our direction (and even better, it can be connected to Cosmos Hub).
+
+We are introducing two new cryptographic technologies with Tendermint-BFT to further improve our blockchain. One is Verifiable Random Function (VRF), which was introduced to randomly select the Proposer node that will generate blocks and makes future selection unpredictable. This randomness is expected to deter malicious attacks and make it difficult for participants to act in collusion at some point in the future.
+
+Another feature is the  Boneh–Lynn–Shacham (BLS) signature. BLS signature schemes, which are based on bilinear mapping, gives us the ability to aggregate multiple digital signatures into a single one. In many blockchain protocols, large amounts of signatures must be stored to approve a block. Enabling BLS signature aggregation reduces the footprint and can significantly improve communication overhead and storage consumption.
```

Comment:
Isn't it singular? Isn't it `BLS signature scheme` instead of `BLS signature schemes`?

Reply:
I see, fix it.